src/Form/AddressBookType.php

```diff
@@ -133,24 +133,29 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
             if (null !== $address) {
 
-                $addresses = $options['with_addresses'];
-                if (!is_array($addresses)) {
-                    if (is_callable([ $addresses, 'toArray' ])) {
-                        $addresses = $addresses->toArray();
+                $addressBook = $options['with_addresses'];
```

Comment:
should be `addressesBook`? (plural)

Reply:
No, it looks correct: https://en.wikipedia.org/wiki/Address_book